src/Lens/LensTypes.sol

```diff
@@ -148,7 +148,6 @@ struct InterestRateInfo {
     uint256 cash;
     uint256 borrows;
     uint256 borrowSPY;
-    uint256 supplySPY;
```

Comment:
Is this an ABI breaking change?

Reply:
as discussed, yes. because supplySPY appears to be a nonsensical value